lib/entropy/server/interfaces/main.py

```diff
@@ -6921,7 +6928,8 @@ def add_packages_to_repository(self, repository_id, packages_data,
             try:
                 # add to database
                 package_id, destination_paths = self._package_injector(
-                    repository_id, package_filepaths, inject = inject)
+                    repository_id, package_filepaths, inject = inject,
+                    reset_revision=reset_revision)
```

Comment:
Breaks convention in the file (x = y).

Reply:
I'm not a clear idea or what means your comment?
Do you think about the mission of the method? That could be with revision 0 or other revision?

Reply:
I mean, someting like this should be done in this case: `reset_revision = reset_revision` not `reset_revision=reset_revision`. Again, to keep consistency.

Reply:
ok, I will do tomorrow this fix